File: Java/ProdukMochi.java
public class ProdukMochi {
    protected String kode, nama;
    protected int harga;

    public ProdukMochi(String kode, String nama, int harga) {
        this.kode = kode;
        this.nama = nama;
        this.harga = harga;
    }

    public String getKode() { return kode; }
    public String getNama() { return nama; }
    public int getHarga() { return harga; }
}

File: CPP/ProdukMochi.cpp
#include "ProdukMochi.h"

ProdukMochi::ProdukMochi(string k, string n, int h) : kode(k), nama(n), harga(h) {}
string ProdukMochi::getKode() { return kode; }
string ProdukMochi::getNama() { return nama; }
int ProdukMochi::getHarga() { return harga; }
